.github/workflows/generate-release.yml

```diff
@@ -47,15 +47,16 @@ jobs:
       ### NOTE: This workflow may time out here while waiting for Mozilla review.
       ### If this occurs, run the "Generate Release (after signing)" action once it is complete.
 
-      # - name: Chrome — Build
-      #   run: npm run build
-      # - name: Chrome — Upload and Publish
-      #   run: npx chrome-webstore-upload --source=web-ext-artifacts/new_xkit-${{ steps.get-version.outputs.result }}.zip
-      #   env:
-      #     EXTENSION_ID: ${{ secrets.CHROME_EXTENSION_ID }}
-      #     CLIENT_ID: ${{ secrets.CHROME_CLIENT_ID }}
-      #     CLIENT_SECRET: ${{ secrets.CHROME_CLIENT_SECRET }}
-      #     REFRESH_TOKEN: ${{ secrets.CHROME_REFRESH_TOKEN }}
+      - name: Chrome — Build
+        run: npm run build
+      - name: Chrome — Upload
+        run: npx chrome-webstore-upload upload --source=web-ext-artifacts/new_xkit-${{ steps.get-version.outputs.result }}.zip
```

Comment:
Typo: should be chrome-webstore-upload-cli (missing -cli)

Reply:
Weirdly, the cli command isn't the same as the package name! https://github.com/fregante/chrome-webstore-upload-cli#usage

Reply:
Yes, but `npx` takes the package name on npm, not the name the package gives to its CLI. Running `npx chrome-webstore-upload` locally gives the error `npm ERR! could not determine executable to run` while with -cli works

Reply:
Oh, interesting! I copied from xkit rewritten, which is using `npx` like what I thought `npm exec` does (runs a command as if it was a package script, relying on dependencies having been installed already). So the comment I was going to make was like, "ah, yeah, I should change this to `npm exec chrome-webstore-upload`; both work so long as you run `npm i` first, but that's more clear that we're not trying to install the (wrong) tool."

But I don't love that either, because in a repository without the tool installed, `npm exec chrome-webstore-upload --version` gives... the npm version. `npm exec` is actually intended for use [in a way I find somewhat odd](https://docs.npmjs.com/cli/v7/commands/npm-exec), mostly because I've become a PNPM user by default and I'm expecting the behavior of `pnpm exec`, from which I got the "runs a command as if it was a package script, relying on dependencies having been installed already" thing.

So it either does make sense here to either use `npx`, and thus stick to the "npx uses the package name and will run the package default command" standard... or, of course, just make a package script, which is probably better.